### PR TITLE
hydra_unstable: 2022-12-05 -> 2022-12-23

### PR DESCRIPTION
**Pull request by NickCao**
###### Description of changes
changelog: https://github.com/NixOS/hydra/compare/14d4624dc20956ec9ff54882e70c5c0bc377921a...f48f00ee6d5727ae3e488cbf9ce157460853fea8
highlight: https://github.com/NixOS/hydra/pull/1256

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by Mindavi — APPROVED** [review]
I think this should be fairly safe. Exiting when there's issues with the DB is a lot nicer than what was happening before, I did run into that before (in OOM situations).

Also, ofborg got it right at once, woo!

File: pkgs/development/tools/misc/hydra/unstable.nix [hunk 1]
```diff
@@ -126,13 +126,13 @@ let
 in
 stdenv.mkDerivation rec {
   pname = "hydra";
-  version = "unstable-2022-12-05";
+  version = "2022-12-23";
```
Comment (SuperSandro2000) [inline pkgs/development/tools/misc/hydra/unstable.nix:129]: Why was that changed back?

**Comment by SuperSandro2000** [comment]
Since this update two test are failing:

```
The following jobs failed:
+--------------------------------------+------------------------------+
| Job ID                               | Test File                    |
+--------------------------------------+------------------------------+
| 694E34DE-84E4-11ED-8232-DC0CE1DAF86B | Hydra/Controller/User/ldap.t |
+--------------------------------------+------------------------------+
                                  Yath Result Summary
----------------------------------------------------------------------------------------
     Fail Count: 1
     File Count: 90
Assertion Count: 1371
      Wall Time: 45.97 seconds
       CPU Time: 510.74 seconds (usr: 0.73s | sys: 0.08s | cusr: 427.82s | csys: 82.11s)
      CPU Usage: 1110%
    -->  Result: FAILED  <--
1..1
not ok 1 - Passed tests when run by yath
yath exited with 256FAIL: test.pl
===================
1 of 2 tests failed
===================
```

**Comment by Mindavi** [comment]
Yeah, it's a known issue that the ldap test is flaky.
